### PR TITLE
Redesign molecule Qcm && add coverage

### PR DESCRIPTION
**Pull request by StefanoA1**
**Detailed purpose of the PR**

- Redesigns molecule Qcm && adds coverage:
  -  Border radius, shadow, font colors, background colors, hover color

- Figma: 
https://www.figma.com/file/IRsndRbOzU38iMZ8LbVWEo/MOOC-2022?node-id=1679%3A3330

**Result and observation**

- Before

Screenshot 2022-02-08 at 15 44 12

- After

Screenshot 2022-02-08 at 17 16 28

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [x] Unit testing

### CODE REVIEW

**Comment by StefanoA1** [comment]
[Re-design composant question QCM](https://trello.com/c/yUxfVGRg/2477-re-design-composant-question-qcm)

**Review by Fausto95 — CHANGES_REQUESTED** [review]

**Review by esa-coorp — APPROVED** [review]

File: packages/@coorpacademy-components/src/molecule/questions/qcm/style.css [hunk 1]
```diff
@@ -18,14 +17,14 @@
...
-  border: 1px solid color(black lightness(85%));
-  border-radius: 3px;
+  border-radius: 8px;
   box-sizing: border-box;
   margin-bottom: 8px;
   cursor: pointer;
   padding: 11px 24px;
   line-height: 24px;
   white-space: normal;
+  box-shadow: 0px 4px 16px rgba(0, 0, 0, 0.12);
```
[the remark this answers is not in the document]
Comment (Fausto95) [reply packages/@coorpacademy-components/src/molecule/questions/qcm/style.css:27]: Actually, just review all the designs, and this box-shadow is the same across multiple elements.
Mind creating a variable for this? One could name it: revisionBoxShadow

**Review by Fausto95 — CHANGES_REQUESTED** [review]

File: packages/@coorpacademy-components/src/molecule/questions/qcm/style.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (StefanoA1) [reply packages/@coorpacademy-components/src/molecule/questions/qcm/style.css:27]: @Fausto95 for the rgba color or the whole variable? si couleur j'ai trouvé que: notification-banner et popin qui utilisent ça

File: packages/@coorpacademy-components/src/molecule/questions/qcm/index.js [hunk 2]
```diff
@@ -1,42 +1,73 @@
...
-    );
-  });
+        const hoverReplaceActions = selected
+          ? {
+              // can't access dynamic skin color on :hover, so an artificial
+              // hover has to be made for selected answers
+              onMouseEnter: () => dispatch([answerKey, true]),
+              onMouseLeave: () => dispatch([answerKey, false])
+            }
+          : {};
```
Comment (StefanoA1) [inline packages/@coorpacademy-components/src/molecule/questions/qcm/index.js:52]: sur unselected on laisse le css gérer le hover car on a pas besoin de couleurs de skin pour ça

**Review by Fausto95 — APPROVED** [review]

**Review by esa-coorp — APPROVED** [review]

**Pull request merged**